### PR TITLE
Dev arun

### PR DESCRIPTION
**Pull request by arunbodd**
Following have been changed/updated from the last commit:

   1. Removed Kraken and Pangolin variant estimators from the pipeline
   2. Accounted for minimap2 long reads and single-end/paired-end illumina reads
   3. Updated the README file - Significant changes have been made, so please follow along
   4. Added fastqc and fastp modules accounting for long reads
   5. Input_check has the option to use "only" bam file as input for Iontorrent, however, a subworkflow need to be developed. Also,
if iontorrent has fastq as input, the current pipeline doesn't account for it.
   6. minimap2 parameters change dynamically depending on the sequencing platform

*All examples used in test samplesheet.csv are from nf-core/testdatasets.

### CODE REVIEW

File: assets/samplesheet_with_nanopore.csv
Comment (hseabolt) [inline assets/samplesheet_with_nanopore.csv:1]: Need to update to remove hard-coded paths on specific compute systems.

File: assets/samplesheet.csv
Comment (hseabolt) [inline assets/samplesheet.csv:1]: Update file paths to remove hard-coded paths.

File: conf/cdc.config [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+*/
+
+/*
+========================================================================================
+    Default institutional and max resource paramters - subject to overwrite.
+========================================================================================
+*/
+
+params {
+  config_profile_description = 'Rosalind HPC @ CDC'
```
Comment (hseabolt) [inline conf/cdc.config:28]: Remove reference to Rosalind, keep this generic.

File: conf/cdc.config
Comment (hseabolt) [inline conf/cdc.config:1]: Probably remove this config file, as it is specific to CDC HPC and it is available internally to users that have need of it.

File: conf/sge.config
Comment (hseabolt) [inline conf/sge.config:1]: Okay to leave this config as a generic Grid Engine config

File: conf/test.config
Comment (hseabolt) [inline conf/test.config:1]: Remove TODO lines from this file, update any hard-coded filepaths, and set an input file.  Test profiles should include the input file directly.

File: nextflow.config [hunk 2]
```diff
@@ -152,6 +145,7 @@ profiles {
     test      { includeConfig 'conf/test.config'      }
     test_full { includeConfig 'conf/test_full.config' }
     sge	      { includeConfig 'conf/sge.config'       }
+    rosalind  { includeConfig 'conf/cdc.config'  }
```
Comment (hseabolt) [inline nextflow.config]: remove rosalind component, line 148

**Review by hseabolt — COMMENTED** [review]
Minor corrections mainly for style or data privacy needed.

File: nextflow.config (hunk 2 above)
Comment (arunbodd) [reply nextflow.config]: Removed it and resolved this issue

File: conf/test.config
Comment (arunbodd) [reply conf/test.config:1]: Removed these lines in the code

File: conf/sge.config
Comment (arunbodd) [reply conf/sge.config:1]: Yep, removed rosalind and retained sge config

File: conf/cdc.config
Comment (arunbodd) [reply conf/cdc.config:1]: Done

File: assets/samplesheet.csv
Comment (arunbodd) [reply assets/samplesheet.csv:1]: added testdata sets from nf-core/testdatasets

File: assets/samplesheet_with_nanopore.csv
Comment (arunbodd) [reply assets/samplesheet_with_nanopore.csv:1]: Done

**Pull request merged**